Scripts/Python/Neighborhood.py

```diff
@@ -81,3 +81,54 @@ def OnFirstUpdate(self):
             
     def OnNotify(self,state,id,events):
         pass
+
+    def _AddClothingFromAgeSDL(self, clothingList, *sdls):
+        ageSDL = PtGetAgeSDL()
+        for i in sdls:
+            if not ageSDL[i][0]:
+                PtDebugPrint(f"Neighborhood._AddClothingFromAgeSDL(): You don't have the SDL '{i}' so no '{clothingList}' for you", level=kWarningLevel)
+                return
+
+        if isinstance(clothingList, str):
+            clothingList = [clothingList]
+
+        avatar = PtGetLocalAvatar()
+        if avatar.avatar.getAvatarClothingGroup() == kFemaleClothingGroup:
+            clothingPrefix = "FReward"
+        else:
+            clothingPrefix = "MReward"
+
+        entireClothingList = avatar.avatar.getEntireClothingList(0)
+        for i in filter(None, clothingList):
+            desiredName = "_".join((clothingPrefix, i))
+            if desiredName not in entireClothingList:
+                PtDebugPrint(f"ERROR: Neighborhood._AddClothingFromAgeSDL(): Invalid clothing item '{desiredName}' requested!")
+                continue
+            if not any((j[0] == desiredName for j in avatar.avatar.getWardrobeClothingList())):
+                PtDebugPrint(f"Neighborhood._AddClothingFromAgeSDL(): Adding '{desiredName}' clothing item to your closet! Aren't you lucky?", level=kWarningLevel)
+                avatar.avatar.addWardrobeClothingItem(desiredName, ptColor().white(), ptColor().white())
```

Comment:
Will there be a KI system message to notify the user of the new wardrobe item(s)?

Reply:
There isn't currently for the relto age.
I could add it, but it requires a lot of extra code.
Feel like we should probably make the extra code a helper function instead.

Reply:
I didn't realize it wasn't already a shared/helper function. I am 👍 on refactoring and using shared code. Probably in a separate PR would be best, though.